pkg/container-hook/tracer.go

```diff
@@ -560,6 +560,16 @@ func (n *ContainerNotifier) monitorRuntimeInstance(bundleDir string, pidFile str
 		return fmt.Errorf("marking %s: %w", configJSONPath, err)
 	}
 
+	// similar to config.json, we ignore passwd file if it exists
+	passwdPath := filepath.Join(bundleDir, "passwd")
+	if _, err := os.Stat(passwdPath); err == nil {
```

Comment:
With `config.json`, we test the error with `errors.Is(err, os.ErrNotExist)` but with `passwd`, we test the error with `err == nil`. Is there a reason for testing differently? Ditto in runcfanotify.

Reply:
Done. `passwd` file is optional compared to `config.json` but makes sense to use `!errors.Is(err, os.ErrNotExist)` for clarity and let it return an `error` in case it happens in following calls.